### ISSUE TITLE
Data export issues 

### ISSUE THREAD

**Issue by hotavocado** [ChildMindInstitute/mindlogger-admin-OLD-Vue#901]
<img width="1274" alt="Screen Shot 2021-04-02 at 7 51 42 PM" src="https://user-images.githubusercontent.com/24463939/113462173-9e760900-93ed-11eb-8432-1a3677d0c62c.png">

1. activity_scheduled_time, activity_start_time, activity_end_time columns are not working as intended:

+  activity_scheduled_time should show YYYY-MM-DD-21:00 timestamps (this activity is has daily repeat schedule of 9:00pm)

+ activity_start_time/activity_end_time should show timestamps instead of NULL

2. Have 'activity' column show [activity name](https://github.com/ChildMindInstitute/DBT-pilot/blob/6b551d308284312c1c6fc729cf3de41e05287c1a/activities/dbt_daily/dbt_daily_schema#L7) (take out _schema part) instead of token

3. Export items in order of the [order object in the activity schema](https://github.com/ChildMindInstitute/DBT-pilot/blob/6b551d308284312c1c6fc729cf3de41e05287c1a/activities/dbt_daily/dbt_daily_schema#L116-L136) instead of alphabetically

**Comment by sergeysemenuta** [comment]
Activity scheduled time always shows "not scheduled"
The activity column still shows tokens 
![2021-04-07_18-39-56.jpg](https://images.zenhubusercontent.com/5e59389dceb759b87367c8cb/32202561-d821-4700-ac52-1f16166afe4a)
Checked on staging
https://admin-staging.mindlogger.org/

**Comment by hotavocado** [comment]
Thanks @sergeysemenuta, I just want to clarify:

For the 'activity' column, originally this would have a github URL for applets imported from github and a token for applets created/edited with the applet editor. Now data export from all applets have tokens.

Minji and I depended on the github URL to pull activity jsons into R to generate reports (specifically the item order object). Is there is a way to access the same activity/item jsons using the tokens provided in this current export? If so we need to make sure these tokens stay in the data export.

I suggested replacing the 'activity' column with the activity name string in my original post, but I forgot to mention that we should also keep the tokens (or some information that allows the user of the data export to access the applet schemas) as well. If the tokens work like links to the schema files, we can create 2 new columns:

1. activity_schema (token that links to the activity schema)
2. item_schema (token that links to the item schema)

I realize info from many columns inside this export come directly from the item/activity schema, and it might seem redundant to include the info as strings and also provide the link to the schema. The reason for this is to make commonly used info more easily accessible so that analysts (many who are used to working only with tabular data formats like CSV) don't have to go through the extra hassle of importing the applet jsons to access info like activity name, item name, question, choices, etc. However, having the links to the jsons is also important so that people who want to access the data in this format can do so easily using the data export, without having to do a secondary SQL dump or something like that..

**Comment by sergeysemenuta** [comment]
The issues are fixed
![2021-04-09_13-07-22.jpg](https://images.zenhubusercontent.com/5e59389dceb759b87367c8cb/38a2b59d-63d6-41f6-81b1-db6a8f92bcf1)
Checked on: 
https://admin-staging.mindlogger.org/

**Comment by jj105** [comment]
@hotavocado

just to confirm,

for activities/items that are imported from github we need to export github link.
and for activities/items created from builder, we need to export link to somewhere else ( backend or admin panel ) so that user will be able to view item json ( similar to github ) once he clicks link.

is that correct?

**Comment by hotavocado** [comment]
@jj105 yes exactly! we just need some kind of link that can take the user to the raw json for [activity_schema](https://raw.githubusercontent.com/ChildMindInstitute/DBT-pilot/main/activities/dbt_daily/dbt_daily_schema) and [item](https://raw.githubusercontent.com/ChildMindInstitute/DBT-pilot/main/activities/dbt_daily/items/anger) so they can view/download it

**Comment by jj105** [comment]
@hotavocado

what if we provide two different interfaces for that?

1 - viewing single item via link ( the structure of data will be similar to [item](https://raw.githubusercontent.com/ChildMindInstitute/DBT-pilot/main/activities/dbt_daily/items/anger)  )
2 - downloading whole schema ( just like a format for downloading content of repo from github as zip file )

**Comment by jj105** [comment]
by the way, should links give sense about what the content is for?

https://raw.githubusercontent.com/ChildMindInstitute/DBT-pilot/main/activities/dbt_daily/items/anger
For example for above link, reviewer might know that  it is link for an item inside DBT pilot applet dbt daily activity without opening it.

or is it fine to create/export random url ( domain will still be mindlogger.org ) with correct content in it?

**Comment by hotavocado** [comment]
@jj105 

1- yep perfect, would be very useful to have these links for items and activity schema in data export for easy access
2 - yes, i was wondering when this would be possible! there is an "export schema" button the admin panel but it's not functional right now. we can give this link to the button, no need to include it in data export.
![Screen Shot 2021-04-21 at 4 36 35 PM](https://user-images.githubusercontent.com/24463939/115617955-07032800-a2c0-11eb-9828-7f06367f7407.png)

it would be nice for the links in the data export to look meaningful like the ones from github, but if this will take lots of time a random url with correct content is totally fine

**Comment by jj105** [comment]
ok. thank you for your explanation!
